netbox/virtualization/forms/filtersets.py

```diff
@@ -154,11 +154,19 @@ class VirtualMachineFilterForm(
         choices=VirtualMachineStatusChoices,
         required=False
     )
+    manufacturer_id = DynamicModelMultipleChoiceField(
+        queryset=Manufacturer.objects.all(),
+        required=False,
+        label=_('Manufacturer'),
+    )
```

Comment:
There is no `manufacturer_id` filter for virtual machines, so this form field would have no effect beyond constraining the choices for the platform field below, which could be very confusing to the user. The scope of FR #12741 deals only with the VM & device edit forms; we shouldn't be making any changes to the list filters as part of this work anyway.